tests/test_yamls/intermediate_bucket.yaml

```diff
@@ -0,0 +1,14 @@
+name: intermediate-bucket
+
+file_mounts:
+  file_mounts_dir: .
+
+workdir: .
+
+
+setup: |
+  echo "running setup"
+
+run: |
+  conda env list
+  echo "task run finish"
```

Comment:
Should this test try to `ls .` (for workdir) and `ls /file_mounts_dir` to verify if file_mounts were successful? Refer to using_file_mounts.yaml for an example..